source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,48 @@
+.. _railsmdb-get-started-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Dependencies
+
+      Ensure you have the following dependencies installed
+      in your development environment:
+      
+      - `Ruby version 3.0 or later <https://www.ruby-lang.org/en/downloads/>`__
+      - `RubyGems <https://rubygems.org/pages/download>`__
```

Comment:
I don't think we need to list RubyGems as a dependency; it's been shipped with Ruby since Ruby 1.9; I'm not even sure why there's a download link for RubyGems, except maybe to update an old installation? Regardless, I don't think we need to list it here.

Reply:
Good to know! Removed